docs/spec/src/protocol/payments/payment_system.md

```diff
@@ -0,0 +1,178 @@
+# EigenDA Payment System
+
+## 1. Overview
+
+The EigenDA payment system allows users to pay for blob dispersals through two methods: reservations and on-demand
+payments. All payment logic is implemented in the [`core/payments`](../../../../../core/payments/) package.
+
+**Key Concepts:**
+- Blob sizes are measured in *symbols*, where each symbol contains 32 bytes of data
```

Comment:
nit: each symbol contains 31 bytes of payload data. Not sure if worth mentioning here though..

Reply:
b279c02ab587272e8b5e6733ec8807f59e0d64ba

tweaked wording